### PR TITLE
Fix: Update COMP triage accept screens, reject option

### PR DESCRIPTION
**Pull request by robertdeniszczyc2**
The reject to CCH option had been set with the value CCH This broke existing cases as the old workflows expected a Yes/No value This change updates the CCH value to be no, so it becomes backwards compatible

### CODE REVIEW

File: src/main/resources/processes/COMP/COMP_EXGRATIA_TRIAGE.bpmn [hunk 1]
```diff
@@ -163,7 +163,7 @@
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${CctTriageAccept == "Yes"}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
     <bpmn:sequenceFlow id="SequenceFlow_0yakx66" sourceRef="ExclusiveGateway_1hbd7xu" targetRef="Screen_Transfer">
-      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${CctTriageAccept == "No" || CctTriageAccept == "CCH"}</bpmn:conditionExpression>
+      <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${CctTriageAccept == "No"}</bpmn:conditionExpression>
```
Comment (jeff-horton-ho-sas) [inline src/main/resources/processes/COMP/COMP_EXGRATIA_TRIAGE.bpmn:166]: Should CCH be left in so we can eventually switch back to CCH once old cases are cleared out?

**Review by jeff-horton-ho-sas — APPROVED** [review]
One comment, but happy to accept as is if you disagree.

Do we also need to update the label for this value in [hocs-casework](https://github.com/UKHomeOffice/hocs-casework/blob/main/src/main/resources/config/details/stages/COMP.json#L972)?

**Comment by robertdeniszczyc2** [comment]
Converted PR to Draft while we confirm impact of outstanding questions

**Comment by LiamMacP** [comment]
Moved to Ready as business confirmed they are happy with the change - HOCS-6603

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**